StreamChatUITestsAppUITests/Tests/PushNotification_Tests.swift

```diff
@@ -6,125 +6,245 @@ import XCTest
+                             mutableContent: "test",
+                             category: 42,
+                             type: 42,
+                             sender: 42,
+                             version: 42,
+                             messageId: 42,
+                             cid: 42)
         
-        try XCTSkipIf(ProcessInfo().operatingSystemVersion.majorVersion < 14,
-                      "Remote push notifications do not work on iOS < 14")
+        GIVEN("user goes to message list") {
+            userRobot.login().openChannel()
+        }
+        checkHappyPath(message: message, sender: app.label.uppercased())
+    }
+    
+    func test_pushNotification_optionalValuesContainIncorrectData() throws {
+        linkToScenario(withId: 295)
+
+        mockPushNotification(body: message,
+                             title: -1,
+                             badge: -1,
+                             mutableContent: -1,
+                             category: "test",
+                             type: "test",
+                             sender: "test",
+                             version: "test",
+                             messageId: "test",
+                             cid: "test")
         
-        let message = "hey user#1 👋"
+        GIVEN("user goes to message list") {
+            userRobot.login().openChannel()
+        }
+        checkHappyPath(message: message, sender: app.label.uppercased())
+    }
+    
+    func test_pushNotification_requiredValuesAreInvalid() throws {
+        linkToScenario(withId: 296)
 
-        GIVEN("user opens the channel") {
-            userRobot
-                .login()
-                .openChannel()
+        GIVEN("user goes to message list") {
+            userRobot.login().openChannel()
         }
         AND("user goes to background") {
             deviceRobot.moveApplication(to: .background)
         }
-        AND("participant sends a message") {
-            participantRobot.wait(1).sendMessage(message,
+        
+        mockPushNotification(body: nil)
+        WHEN("participant sends a message (push body param is nil)") {
+            participantRobot.wait(2).sendMessage("\(message)_0",
                                                  withPushNotification: true,
                                                  bundleIdForPushNotification: app.bundleId())
         }
-        WHEN("user taps on the push notification") {
-            userRobot.tapOnPushNotification()
+        THEN("user does not receive a push notification") {
+            userRobot.assertPushNotificationDoesNotAppear()
+        }
+        
+        mockPushNotification(body: "")
+        WHEN("participant sends a message (push body param is empty)") {
+            participantRobot.sendMessage("\(message)_1",
+                                                 withPushNotification: true,
+                                                 bundleIdForPushNotification: app.bundleId())
```

Comment:
Same formatting issue, we try to avoid using the ObjC old style of formatting 👍

Reply:
Thanks, fixed everywhere 👍